src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -46,9 +46,9 @@
     <_pastShimFiles Include="System.Net.Security.Native.so" />
     <_pastShimFiles Include="System.Security.Cryptography.Native.Apple.a" />
     <_pastShimFiles Include="System.Security.Cryptography.Native.Apple.dylib" />
-    <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.a" />
-    <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.dylib" />
-    <_pastShimFiles Include="System.Security.Cryptography.Native.OpenSsl.so" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.Android.a" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.Android.dylib" />
+    <_pastShimFiles Include="System.Security.Cryptography.Native.Android.so" />
```

Comment:
This is not Android specific, should stay unchanged.

Reply:
Ah got it. I wasn't sure on the extent of renaming. Thank you for helping me catch all the non-Android specific ones.